### PR TITLE
return 204 and empty string if request.data is empty

### PR DESCRIPTION
**Pull request by aW3st**
Would like to add a test for this but not very familiar with testing flask apps.

### CODE REVIEW

File: predictor.py [hunk 1]
```diff
@@ -142,8 +142,7 @@ def ping():
 @app.route('/invocations', methods=['POST'])
 def transformation():
     if not request.data:
-        return Response(response='empty payload',
-                        status=415, mimetype='text/plain')
+        return Response(response='', status=204)
```
Comment (shshe) [inline predictor.py:145]: Does Viaduct handle this empty response well? @adamklec mentioned that this route returns an empty list for the `ner` version and it should be a list of tuples for the `classify` cas.

Comment (aW3st) [reply predictor.py:145]: I think you're right--this should be empty list.

Comment (adamklec) [reply predictor.py]: I think classify should always be a list of 2 numbers at this point (the probability of the positive and negative class). I believe we only support binary classification right now.

File: predictor.py [hunk 2]
```diff
@@ -142,8 +142,7 @@ def ping():
 @app.route('/invocations', methods=['POST'])
 def transformation():
     if not request.data:
-        return Response(response='empty payload',
-                        status=415, mimetype='text/plain')
+        return Response(response=[], status=204)
```
Comment (adamklec) [inline predictor.py:145]: Interesting use of the "204 No Content" response code. It makes sense.

**Review by adamklec — APPROVED** [review]

File: predictor.py (hunk 2 above)
Comment (aW3st) [reply predictor.py:145]: as usual, credit to [stack overflow](https://stackoverflow.com/questions/24295426/python-flask-intentional-empty-response)

**Pull request merged**